ajax/libs/fuzzyset.js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "fuzzyset.js",
+  "description": "A fuzzy string set for JavaScript",
+  "version": "0.0.1",
+  "homepage": "https://glench.github.com/fuzzyset.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Glench/fuzzyset.js.git"
+  },
+  "author": "Glen Chiacchieri (http://glench.com/)",
+  "filename": "fuzzyset.min.js",
+  "keywords": [
+    "akin",
+    "search",
+    "string"
+  ],
+  "license": "BSD",
```

Comment:
because the author doesn't provide LICENSE.txt or BSD license content, I can't tell which BSD license he use.(I mentioned it in init comment)